app.py

```diff
@@ -30,18 +30,19 @@ def __repr__(self):
 dict = {}
 def parse_response(json):
     if 'text' in json['message']:
-        match = re.search(r'(\w+):(\w+):(.+)', json['message']['text'])
+        match = re.search(r'(\w+):[^:]:(.+)', json['message']['text'])
         if match is not None:
             if(match.group(1) == "add"):
-                pattern = match.group(2)
+                pattern = str(match.group(2))
                 answer = match.group(3)
-                if len(pattern)> 2:
+                if len(pattern) >= 2:
                     dict[pattern] = answer
                     session.add(Response(pattern,answer))
                     session.commit()
             if match.group(1) == "henry":
                 if match.group(2) == "delete":
                     if match.group(3) in dict.keys():
+                        match.group(3).capitalize()
```

Comment:
Dit is niet meer nodig dacht ik. denk sws niet dat je op deze manier kan capitalize